tools/localazy/config.json

```diff
@@ -65,7 +65,8 @@
         "screen_create_room_.*",
         "screen\\.create_room\\..*",
         "screen_start_chat_.*",
-        "screen\\.start_chat\\..*"
+        "screen\\.start_chat\\..*",
+        "screen_room_directory_search_title*"
```

Comment:
It's OK but the string will be unused in the module it was used before. Ideally the Localazy key should be renamed.

Reply:
No, this key is also used in another screen (the room directory one).
The key used in the previous module will be removed.